engine/src/cmd/collide2/OPC_PlanesCollider.h

```diff
@@ -3,6 +3,8 @@
  *	OPCODE - Optimized Collision Detection
  *	Copyright (C) 2001 Pierre Terdiman
  *	Homepage: http://www.codercorner.com/Opcode.htm
+ *  Copyright (C) 2021, 2022, 2025 Stephen G. Tuggy
+ *  Copyright (C) 2023 Benjamen R. Meyer
```

Comment:
we need to reference a license of some sort for this...
from http://www.codercorner.com/Opcode.htm:
```
You’re free to use them in any commercial or non-commercial program.
```
Which we can probably reference as an `UNLICENSE` or probably best to just say `Public Domain` (or https://wiki.creativecommons.org/wiki/public_domain) as that's effectively what it is.

Reply:
Fixed